packages/cli/package.json

```diff
@@ -54,7 +55,7 @@
   },
   "scripts": {
     "prepublishOnly": "nsp check",
-    "test": "mocha --opts node_modules/@loopback/build/mocha.js.opts \"test/*.js\""
+    "test": "lb-mocha 'test/*.js'"
```

Comment:
Ditto, please preserve original double quotes here and also in all other package scripts you have modified in this patch.

See https://stackoverflow.com/q/10737283/69868 and https://stackoverflow.com/a/24181667/69868